values.yaml

```diff
@@ -52,6 +52,35 @@ global:
   # consul-k8s v0.8.0+.
   bootstrapACLs: false
 
+  # Enables TLS encryption across the cluster to verify authenticity of the
+  # servers and clients that connect. Note: It is HIGHLY recommended that you also
+  # enable Gossip encryption.
```

Comment:
This should really link to our own docs page on enabling gossip encryption... if we had one.